released/packages/coq-rupicola/coq-rupicola.0.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: [
+  "Massachusetts Institute of Technology"
+]
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "https://github.com/mit-plv/rupicola"
+bug-reports: "https://github.com/mit-plv/rupicola/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%" "EXTERNAL_DEPENDENCIES=1" "all" FIXME EXTERNAL_DEPS]
```

Comment:
???

Reply:
This is a note to self that I need to update the tag after https://github.com/mit-plv/rupicola/pull/52 is merged and a new version is tagged

Reply:
@JasonGross is there anything specific blocking progress on the rupicola package/tag? Every other package here looks good to go.

Reply:
I'm waiting for an approval to merge https://github.com/mit-plv/rupicola/pull/52.  I'm not sure if there's anything in particular blocking that; @cpitclaudel ?

Reply:
I've just tagged v0.0.2.  I'm not at a computer right now and don't know how to calculate the sha512 from my phone, but I'll fix the rest of the files now.

Reply:
If someone else can rename the rupicola opam folder so that it's version 0.0.2 instead of 0.0.1, and fix the checksum, the hopefully this will all build.